tasks/us/census/tiger.py

```diff
@@ -664,10 +666,10 @@ def populate(self):
         session = current_session()
         stmt = ('INSERT INTO {output} '
                 'SELECT geoid, ST_Union(ST_MakePolygon(ST_ExteriorRing(the_geom))) AS the_geom, '
-                '       MAX(aland) AS aland, cdb_observatory.FIRST(name) AS name '
+                '       MAX(aland) AS aland '
```

Comment:
I'm not exactly sure about this particular part of the code... does this have to do with the adding of the proper name for certain geometries?